app/src/main/java/com/github/h3lp3rs/h3lp/signin/SignInActivity.kt

```diff
@@ -49,13 +53,22 @@ class SignInActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
         // Store the context for local storage use
         globalContext = this
+
         setContentView(R.layout.activity_sign_in)
         // Initialize Firebase Auth
         findViewById<ImageButton>(R.id.signInButton).setOnClickListener{
             launchSignIn()
         }
+
+        // Continue without sign in
+        findViewById<TextView>(R.id.noSignInText).setOnClickListener {
```

Comment:
I see that we are not completely consistent with the id names we give. We have to agree on either camel or snake case. But it's no big deal, can be refactored in a less important future task

Reply:
You can import R.id.noSignInText instead of using the findViewById method

Reply:
whaaat how

Reply:
black magic ;)